README.md

```diff
@@ -44,6 +44,7 @@ These are to be used via EVMC `set_option`:
 - `metering=true` will enable metering of bytecode at deployment using the [Sentinel system contract] (set to `false` by default)
 - `evm1mode=<evm1mode>` will select how EVM1 bytecode is handled
 - `sys:<alias/address>=file.wasm` will override the code executing at the specified address with code loaded from a filepath at runtime. This option supports aliases for system contracts as well, such that `sys:sentinel=file.wasm` and `sys:evm2wasm=file.wasm` are both valid. **This option is intended for debugging purposes.**
+- `fuzzing=true` will enable fuzzing mode, which aborts on an internal error (as opposed to transmitting the error code to the client). **This option is solely intended for fuzzing purposes.**
```

Comment:
How about naming it `abort-on-internal-error`?

Reply:
Any shorter suggestion?

Reply:
No.

Reply:
A suggestion: "errormode", with possible value "abort-on-internal-error".